src/chapter3/section4/Exercise2.java

```diff
@@ -161,7 +161,6 @@ public void put(Key key, Value value) {
             }
 
             keysSize++;
-            node = buckets.get(bucketIndex);
```

Comment:
We need this line because the `node` variable may be null after going over the loop at line 154.
Also note that when deleting this line the tests fail.

Reply:
> We need this line because the `node` variable may be null after going over the loop at line 154.
> Also note that when deleting this line the tests fail.

i see...my bad，thank you for response